themes/theme-gmd/pages/Cart/components/Item/components/Coupon/components/Layout/components/Code/index.jsx

```diff
@@ -1,18 +1,27 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import I18n from '@shopgate/pwa-common/components/I18n';
+import { Ellipsis } from '@shopgate/engage/components';
+import { css } from 'glamor';
+
+const styles = {
+  ellipsis: css({
+    whiteSpace: 'nowrap',
+    display: 'inline-block !important',
+  }).toString(),
+};
 
 /**
 * The Coupon Code component.
 * @param {Object} props The component props.
 * @returns {JSX}
 */
 const Code = ({ value }) => (
-  <div>
+  <Ellipsis className={styles.ellipsis}>
```

Comment:
default prop of component is rows: 3.
i guess we should better set rows to 1 here